smoke-tests/spec/fixtures/privileged-deployment.yaml.erb

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
```

Comment:
@jasonBirchall , can we use apps/v1 as we are moving towards k8s 1.16

Reply:
A very good point. I'll change both. :+1: